server/app/playbook_run_service.go

```diff
@@ -1566,7 +1566,10 @@ func (s *PlaybookRunServiceImpl) DuplicateChecklistItem(playbookRunID, userID st
 	checklistItem := playbookRunToModify.Checklists[checklistNumber].Items[itemNumber]
 	checklistItem.ID = ""
 
-	playbookRunToModify.Checklists[checklistNumber].Items = append(playbookRunToModify.Checklists[checklistNumber].Items, checklistItem)
+	playbookRunToModify.Checklists[checklistNumber].Items = append(
+		playbookRunToModify.Checklists[checklistNumber].Items[:itemNumber+1],
+		playbookRunToModify.Checklists[checklistNumber].Items[itemNumber:]...)
+	playbookRunToModify.Checklists[checklistNumber].Items[itemNumber] = checklistItem
```

Comment:
Trying this out in the Go playground, this seems to put the new element before the item to be duplicated, instead of after. It probably doesn't matter due to the items being the same, but something may change about this logic later that could cause issues.

This worked for me:

```suggestion
	playbookRunToModify.Checklists[checklistNumber].Items = append(
		playbookRunToModify.Checklists[checklistNumber].Items[:itemNumber+1],
		playbookRunToModify.Checklists[checklistNumber].Items[itemNumber:]...)
	playbookRunToModify.Checklists[checklistNumber].Items[itemNumber+1] = checklistItem
```

Reply:
Updated this.